nixos/modules/services/misc/sssd.nix

```diff
@@ -28,5 +33,10 @@ in {
     };
 
     system.nssModules = optional cfg.enable pkgs.sssd;
+    services.nscd = {
+      enable = mkForce true;
```

Comment:
I think we shouldn't use `mkForce`; rather, add an `assertion` that it's enabled. It is anyway by default but this way if user disables it for some reason they'd know of problems explicitly and can choose what to do.